server.js

```diff
@@ -113,9 +119,17 @@ app.get('*', (req, res, next) => {
         const pluginSiteApiVersion = store.getState().data.info.commit.substring(0, 7);
         const reduxState = JSON.stringify(store.getState()).replace(/</g, '\\x3c');
         const pluginNotFound = req.url !== '/' && store.getState().ui.plugin === null;
+
+        const title = store.getState().ui.plugin && store.getState().ui.plugin.title ? `${store.getState().ui.plugin.title} - Jenkins plugin` : defaultPluginTitle;
+        const description = store.getState().ui.plugin && store.getState().ui.plugin.excerpt ? store.getState().ui.plugin.excerpt : defaultPluginDescription;
+        const opengraphImage = defaultPluginOpenGraphImage; // TODO add support for plugins to provide their own OG imag
```

Comment:
Add a ticket to https://issues.jenkins-ci.org/browse/WEBSITE-637 ?

Reply:
There already was one, added reference